data_structures/check_binary_search_tree.py

```diff
@@ -74,85 +76,80 @@ def _display_aux(self):
         return lines, n + m + u, max(p, q) + 2, n + u // 2
 
 
-class Binary_tree:
+class BinartTree:
     def __init__(self):
```

Comment:
Why not have pylist as an input to the `__init__`

Reply:
py_list added as a input parameter